moon/controller.py

```diff
@@ -8,6 +8,8 @@
 
 from datetime import timedelta
 from statistics import median
+from shapely.geometry import Point, LineString
+from shapely.geometry.polygon import Polygon
```

Comment:
this `import Polygon from polygon` is strange - in controller2 you import it directly from geometry